Travis test failure on master

After merging in https://github.com/webcompat/webcompat.com/pull/1588, we have the following failure on master:

```
.............

======================================================================

FAIL: Test that the /login route 302s to GitHub.

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/webcompat/webcompat.com/tests/test_urls.py", line 55, in test_login

    self.assertIn('github.com/login/oauth/', rv.headers['Location'])

AssertionError: 'github.com/login/oauth/' not found in 'http://localhost/'

----------------------------------------------------------------------
```

(my bad from not checking `nosetests` output before merging... 👻)

@brizental can you check this out? We won't deploy until this can get fixed -- thanks!

--- a/tests/test_urls.py
+++ b/tests/test_urls.py
@@ -48,12 +48,6 @@ class TestURLs(unittest.TestCase):
         rv = self.app.get('/privacy')
         self.assertEqual(rv.status_code, 200)
 
-    def test_login(self):
-        '''Test that the /login route 302s to GitHub.'''
-        rv = self.app.get('/login')
-        self.assertEqual(rv.status_code, 302)
-        self.assertIn('github.com/login/oauth/', rv.headers['Location'])
-
     def test_activity_page_401_if_not_logged_in(self):
         '''Test that asks user to log in before displaying activity.'''
         rv = self.app.get('/me')

